.github/workflows/nightly-jdk11.yml

```diff
@@ -0,0 +1,43 @@
+name: Nightly jkd11
+
+on:
+  schedule:
+    - cron:  '0 6 * * *'
```

Comment:
I don't think we need to separate the corn jobs now, they can run at the same time